kernel/arch/dreamcast/kernel/irq.c

```diff
@@ -326,7 +403,7 @@ irq_context_t *irq_get_context(void) {
 void irq_create_context(irq_context_t *context, uint32_t stkpntr,
                         uint32_t routine, const uint32_t *args, bool usermode) {
     /* Clear out all registers. */
-    memset(context, 0, sizeof(irq_context_t));
+    bzero(context, sizeof(irq_context_t));
```

Comment:
Hadn't you ended up at the conclusion that bzero wasn't an improvement over memset 0 after all? This isn't used in a critical path at all right? Unless it was and there was a huge gain from bzero, I'd suggest against it.

Basically the same for the bzeros in irq_init, but init for sure isn't critical.

Reply:
For posterity found the comment: https://discord.com/channels/488332893324836864/614912396460556298/1249539168770592849 Apparently despite appearing more optimized the outputted asm is just worse.

With the bzero's gone I'm pretty sure the strings include can also be removed.